index.js

```diff
@@ -163,6 +163,7 @@ module.exports = (cmd, args, opts) => {
 
 	const processDone = new Promise(resolve => {
 		spawned.on('exit', (code, signal) => {
+			spawned.killed = signal !== null;
```

Comment:
[See here](https://nodejs.org/api/process.html#process_signal_events)

> Note: Windows does not support sending signals, but Node.js offers some emulation with process.kill(), and ChildProcess.kill(). Sending signal 0 can be used to test for the existence of a process. Sending SIGINT, SIGTERM, and SIGKILL cause the unconditional termination of the target process.

I am guessing `signal` will always be `null` on Windows.